app/models/claim/transfer_brain/data_item_collection.rb

```diff
@@ -54,7 +54,9 @@ def valid_case_conclusion_ids(litigator_type, elected_case, transfer_stage_id)
       end
 
       def data_item_for(detail)
-        seek = DataItem.new(detail.slice(:litigator_type, :elected_case, :transfer_stage_id, :case_conclusion))
+        seek = DataItem.new(detail.slice(:litigator_type, :elected_case, :transfer_stage_id, :case_conclusion, :claim))
+        # TODO: Log this (or remove completely) instead of raising an error
```

Comment:
Do we want to address this TODO now? (Personally I'm happy to leave it for now, I don't think it's a blocker to releasing this).